functions/main.py

```diff
@@ -75,8 +68,6 @@ def data_process(json_data):
         output.append({"text": text, "text_type": text_type, "creator": creator, "createdTime": createdTime})
     return output
 
-# print(get_list_from_api("酒精"))
-
 # functions-framework --target=hello_method
 # curl -X POST -H "Content-Type: application/json" -d '{"title" : "tttt"}' localhost:8080
 def hello_method(request):
```

Comment:
please change function name, and please aware that function and method are different